commit cea9389cc94e85d5d5fc705719147bc4ec7f92da
Author: Brad King <brad.king@kitware.com>
Date:   Mon Jun 28 11:42:34 2010 -0400

    FindMPI: Trust mpicc -showme on BlueGene/L
    
    Do not hard-code known BlueGene/L MPI libraries.  We do not know their
    location so the linker cannot find them without the proper -L search
    path.  The MPI compiler tells us about the libraries anyway, and if it
    does not then the user can fix the problem locally by editing the
    MPI_EXTRA_LIBRARY cache entry.

diff --git a/Modules/FindMPI.cmake b/Modules/FindMPI.cmake
index 481b0e9..ca9649a 100644
--- a/Modules/FindMPI.cmake
+++ b/Modules/FindMPI.cmake
@@ -332,12 +332,6 @@ else (MPI_COMPILE_CMDLINE)
   set(MPI_LINK_FLAGS "" CACHE STRING "MPI linking flags")
 endif (MPI_INCLUDE_PATH AND MPI_LIBRARY)
 
-# on BlueGene/L the MPI lib is named libmpich.rts.a, there also these additional libs are required
-if("${MPI_LIBRARY}" MATCHES "mpich.rts")
-   set(MPI_EXTRA_LIBRARY ${MPI_EXTRA_LIBRARY} msglayer.rts devices.rts rts.rts devices.rts)
-   set(MPI_LIBRARY ${MPI_LIBRARY}  msglayer.rts devices.rts rts.rts devices.rts)
-endif("${MPI_LIBRARY}" MATCHES "mpich.rts")
-
 # Set up extra variables to conform to
 if (MPI_EXTRA_LIBRARY)
   set(MPI_LIBRARIES ${MPI_LIBRARY} ${MPI_EXTRA_LIBRARY})
